Vechain-EasyA-Hackathon-Boston-Oct-2023/milestone-deliveries/Cypress-1.md

```diff
@@ -0,0 +1,26 @@
+| 3.  | Outreach |[Research Presentation](https://www.canva.com/design/DAF6cfFuufM/vcdMTYG8sdO6qLO6cJ7nWA/edit)| Contacted potential investors/partners such as Uber, BLUEBIKES, MBTA to see if a partnership is possible. Slides 11-14|
+| 4.  | Designs |[Competitive Analysis + UX/UI Details](https://www.figma.com/file/xYG6HZh7j5H0acJYD8oKcr/Cypress-Competitive-Analysis?type=whiteboard&node-id=0-1&t=PGb3V4NISAxEFM49-0) | Competitive analysis performed and refined the current UI/UX system based on market research and necessary requirements set out by consumers and brands.| 
+| 5.  | Backend Development |[Images located here](https://docs.google.com/document/d/1G2ePQPoRpvChe3VCsnSWnrCHJGOV9wLnr72aPklMcrU/edit) + [Github Repo for frontend located here, contains backend Web3Auth integration](https://github.com/cypress-labs/leegal)| Created custom user storage using Firebase Firestore, created Web3Auth Connection to Vechain Testnet| 
+| 6.  | Backend |[Go-based backend here](https://github.com/cypress-labs/backend/tree/main)| Set up the Go-based backend environment and version control system.| 
+| 7.  | Integration |[Demo Here](https://drive.google.com/file/d/17fu7obses7gZM2kR1pGqfb1ovSWXJpaY/view) |  This demo shows API endpoints are set up and tested. Integrates backend + smart contract + React Native frontend | 
+| 8.  | Frontend |[Demo Here](https://drive.google.com/file/d/17fu7obses7gZM2kR1pGqfb1ovSWXJpaY/view) + [Repo containing React Native frontend code here](https://github.com/cypress-labs/leegal/tree/main/cypress) + [Figma file containing prototype and added UX/UI pages here](https://www.figma.com/file/llEhoaZHO4GR2qSks42yjW/Cypress?type=design&node-id=26:726&mode=design&t=Np7KkiHSMdNx9ojP-1)| Frontend contains skeleton code for developing the garden management system. Figma contains hi-fidelity versions of what the garden management system will look like and improvements to the social system.| 
+| 9.  | Testing |[Updated smart contract supporting gas sponsorship located here](https://github.com/cypress-labs/leegal/blob/main/Contracts/Cypress.sol) + [documentation showing smart contract testing here](https://docs.google.com/document/d/10PIPJRmZqC33H8okxsj35sEwKSOcvxFmW7yBltfW_xw/edit?usp=sharing)| Tested smart contract functionality as shown in demo, in proper sequence to guarantee no  execution errors| 
+| 10.  | Improvements |[Screenshots showing improvements here](https://github.com/cypress-labs/leegal/blob/main/Contracts/Cypress.sol) + [deployed token contract here](https://explore-testnet.vechain.org/accounts/0x419d1de229bae98cfbb1439e3209c8164ca9b2b3/)| Enabled sponsored gas fee support using VeChain's Designated Gas Payer using Vechain.energy + funded testnet wallet for sponsorship. Also deployed smart contract that creates our own native token: $FERT| 
```

Comment:
The first link points to the contract source rather than the screenshots, could you check?

Reply:
apologies: here are the improvements:

https://docs.google.com/document/d/1cVSxOP0wQgoWftz-qZZZXN1tQO4z418hxLhh0g2Y7nw/edit?usp=sharing

Reply:
Please reflect the link in the delivery content.